modules/packages/LinearAlgebra.chpl

```diff
@@ -1194,16 +1193,16 @@ private proc _lu (in A: [?Adom] ?eltType) {
 }
 
 /*
-  Compute an LU factorization of square matrix `A` 
+  Compute an LU factorization of square matrix `A`
   using partial pivoting, such that `A = P * L * U` where P
   is a permutation matrix. Return a tuple of size 2 `(LU, ipiv)`.
-  
-  `L` and `U` are stored in the same matrix `LU` where 
+
+  `L` and `U` are stored in the same matrix `LU` where
   the unit diagonal elements of L are not stored.
-  
-  `ipiv` contains the pivot indices such that row i of `A` 
+
+  `ipiv` contains the pivot indices such that row i of `A`
   was interchanged with row `ipiv(i)`.
-  
+
```

Comment:
Maybe this one's not needed? Clearly it's personal taste, and you can ignore :)

Reply:
I only deleted whitespace, but I'll remove this newline for you :)

Reply:
You are very kind :)